templates/mainapp/index.html

```diff
@@ -7,6 +7,58 @@
 
 
 {% block main %}
+
+<!-- preloader -->
+<div class="preloader-container preloader-display-none">
+    <div class="spinner">
+        <div class="rec">
+            <div class="rect1"><img src="{% static 'mainapp/png/book.svg' %}" alt="book"></div>
+            <div class="rect2"><img src="{% static 'mainapp/png/book.svg' %}" alt="book"></div>
+            <div class="rect3"><img src="{% static 'mainapp/png/book.svg' %}" alt="book"></div>
+            <div class="rect4"><img src="{% static 'mainapp/png/book.svg' %}" alt="book"></div>
+            <div class="rect5"><img src="{% static 'mainapp/png/book.svg' %}" alt="book"></div>
+        </div>
+        <div class="stand"> </div>
+    </div>
+</div>
+<script>
```

Comment:
Add the script [here](https://github.com/Praful932/Kitabe/blob/1d95220c26c9b245bf3e8b10e3d685722a0bdc4c/templates/mainapp/index.html#L418) we already have a script tag.

Reply:
Ok I will